Boop/Boop/scripts/EvalJavascript.js

```diff
@@ -0,0 +1,26 @@
+/**
+     {
+         "api":1,
+         "name":"Eval",
+         "description":"Evaluate Javascript.",
+         "author":"Sebastiaan Besselsen",
+         "icon":"unknown",
+         "tags":"js,script,run"
+     }
+ **/
+
+function main(input) {
+    const script = input.text.replace(/\n\n---[\s\S]*$/, '');
+
+    let output = '';
+    try {
+        output = eval(script);
+        if (typeof output !== 'string') {
+            output = JSON.stringify(output, null, 2);
+        }
+    } catch (e) {
+        output = 'Error: ' + e;
+    }
+
+    input.text = script + "\n\n---\n\n" + output;
```

Comment:
How about putting the output in a javascript comment so that it can be evaluated again? The next evaluation could remove the old output.

Example:

```js
console.log("Hello world");

/* OUTPUT:
Hello world
*/
```

Reply:
That's a _very_ good idea! Probably better than the dashes...

Reply:
My implementation can always run multiple times because it already strips the last output before evaluating. A disadvantage of the comment approach is that the output does not receive syntax highlighting. Alternative suggestion:

```js
[1, 2, 3].reverse();

// Result:

[
  3, 
  2,
  1
]
```

Reply:
I have implemented my suggestion, looking forward to feedback.